Makefile

```diff
@@ -5,17 +5,6 @@ build:
 	@# Builds the ci image lime-elements.
 	docker build --build-arg GH_TOKEN=${GH_TOKEN} --pull -t $(DOCKER_IMAGE) .
 
-.PHONY: lint
-lint:
-	@# Lints all applicable files.
-	docker run --rm -w /lime $(DOCKER_IMAGE) npm run lint
-
-.PHONY: commitlint
-commitlint: HASH=''
-commitlint:
-	@# Lint the commit message of the commit with the given HASH
-	docker run --rm -w /lime $(DOCKER_IMAGE) npx commitlint -f $(HASH)^ -t $(HASH)
```

Comment:
@jensgustafsson Stavros told me that you usually have opinions when he tries to remove make commands related to Docker, so I should sync with you about this 🙂

Reply:
The commitlint command is already run in GitHub Actions instead of in Jenkins. It's using a 3rd party action, and I've actually already removed the dependency on commitlint. I had just forgotten to clean up the makefile. I really don't think we have all that much use for being able to run commitlint locally. I'd be very surprised if anyone but me has done it even once 😄

Reply:
The lint command is moving from Jenkins to GitHub Actions in this PR, but as you can see in the diff, I haven't used any 3rd party actions for the linting. The workflow is just running the scripts we already have defined in our package.json, since we need to run those locally all the time anyway.